superduper/components/schema.py

```diff
@@ -3,21 +3,38 @@
 
 from overrides import override
 
+from superduper.base.leaf import Leaf
 from superduper.components.component import Component
 from superduper.components.datatype import DataType
 from superduper.misc.reference import parse_reference
 from superduper.misc.special_dicts import SuperDuperFlatEncode
 
-SCHEMA_KEY = '_schema'
 
+class FieldType(Leaf):
```

Comment:
A unified data type identifier used to record native datatypes, replacing the original dtype in Ibis